device/transport/amqp/devdoc/amqp_twin_receiver_requirements.md

```diff
@@ -1,11 +1,11 @@
-# AmqpTwinReceiver Requirements
+# AmqpTwinClient Requirements
 
 ## Overview
 Object used to subscribe to the Cloud-to-Device messages for Twin
 
 ## Example
 ```javascript
-var receiver = new AmqpTwinReceiver(config: ClientConfig, client: any);
+var receiver = new AmqpTwinClient(config: ClientConfig, client: any);
```

Comment:
>AmqpTwinClient( [](start = 19, length = 15)

in the version string for the sdk blah/x.y.z

What gets updated subsequent to the change? x, y, or z?

Reply:
This type is not exported outside of the module. doesn't make any difference for any dependency

---
In reply to: [140019410](https://github.com/Azure/azure-iot-sdk-node/pull/85#discussion_r140019410) [](ancestors = 140019410)